cmd/bucket-lifecycle.go

```diff
@@ -70,6 +71,28 @@ func NewLifecycleSys() *LifecycleSys {
 	return &LifecycleSys{}
 }
 
+func ilmTrace(startTime time.Time, duration time.Duration, oi ObjectInfo, event string) madmin.TraceInfo {
+	return madmin.TraceInfo{
+		TraceType: madmin.TraceILM,
+		Time:      startTime,
+		NodeName:  globalLocalNodeName,
+		FuncName:  event,
+		Duration:  duration,
+		Path:      pathJoin(oi.Bucket, oi.Name),
```

Comment:
Can we include `oi.VersionID` into `madmin.TraceInfo.Path` like so?
```suggestion
		Path:      fmt.Sprintf("%s?versionId=%s", pathJoin(oi.Bucket, oi.Name), oi.VersionID)
```

Reply:
I added a `map[string]string` that allows adding custom fields. That would make it easier to parse.